src/oauth/views.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth import login
+from rest_framework import status
+from rest_framework.response import Response
+from social_django.utils import psa
+from rest_framework.decorators import api_view
+
+@psa('social:complete')
+@api_view(['GET'])
+def login_with_access_token(request, backend):
+    """
+    Social media login
+    """
+    token = request.GET.get('access_token')
+
+    try:
+        user = request.backend.do_auth(token)
+    except Exception as e:
+        return Response(status=status.HTTP_400_BAD_REQUEST)
+
+    if user:
+        login(request, user)
+        return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
Id say 204 is correct in this scenario, if its 200 I would expect there to be something in the response